specification/netapp/resource-manager/Microsoft.NetApp/stable/2019-08-01/netapp.json

```diff
@@ -0,0 +1,2415 @@
+              }
+            ]
+          },
+          "example": "Ultra",
+          "default": "Premium"
+        },
+        "usageThreshold": {
+          "title": "usageThreshold",
+          "type": "integer",
+          "format": "int64",
+          "description": "Maximum storage quota allowed for a file system in bytes. This is a soft quota used for alerting only. Minimum size is 100 GiB. Upper limit is 100TiB. Specified in bytes.",
+          "minimum": 107374182400,
+          "maximum": 109951162777600,
+          "default": 107374182400,
+          "example": 107374182400
+        },
+        "exportPolicy": {
+          "title": "exportPolicy",
+          "description": "Set of export policy rules",
+          "properties": {
+            "rules": {
+              "title": "Export policy rule",
+              "description": "Export policy rule",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/exportPolicyRule"
+              }
+            }
+          }
+        },
+        "protocolTypes": {
+          "title": "protocolTypes",
+          "description": "Set of protocol types",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "example": [
+            "NFSv4.1"
+          ]
+        },
+        "provisioningState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure lifecycle management"
+        },
+        "snapshotId": {
+          "title": "Snapshot ID",
+          "type": "string",
+          "description": "UUID v4 or resource identifier used to identify the Snapshot.",
+          "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}|(\\\\?([^\\/]*[\\/])*)([^\\/]+)$",
+          "maxLength": 36,
+          "minLength": 36,
+          "example": "9760acf5-4638-11e7-9bdb-020073ca3333"
+        },
+        "baremetalTenantId": {
+          "title": "Baremetal Tenant ID",
+          "type": "string",
+          "readOnly": true,
+          "description": "Unique Baremetal Tenant Identifier.",
+          "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+          "maxLength": 36,
+          "minLength": 36,
+          "example": "9560acf5-4e3a-12e7-9bdb-02007cca7779"
+        },
+        "subnetId": {
+          "type": "string",
+          "description": "The Azure Resource URI for a delegated subnet. Must have the delegation Microsoft.NetApp/volumes"
+        },
+        "mountTargets": {
+          "title": "mountTargets",
+          "description": "List of mount targets",
+          "items": {
+            "$ref": "#/definitions/mountTargetList"
+          }
+        },
+        "volumeType": {
+          "type": "string",
+          "description": "What type of volume is this",
+          "example": "DataProtection"
+        },
+        "dataProtection": {
```

Comment:
>dataProtection [](start = 9, length = 14)

Is this specific only to DataProtection volume type? What happens with other volume types?

Reply:
Sounds good.